test/winter_onboarding_2021/fleet_management/views/layout_test.clj

```diff
@@ -0,0 +1,17 @@
+(ns winter-onboarding-2021.fleet-management.views.layout-test
+  (:require [clojure.test :refer [deftest is testing]]
+            [clojure.string :as string]
+            [winter-onboarding-2021.fleet-management-service.views.layout :as layout]))
+
+(deftest application
+  (testing "Should contain bootstrap files & the page title"
```

Comment:
Why not use functions from hiccup find?

Reply:
`layout/application` always returns us HTML string because of the `html5` function used inside it.